doc/RecordingControl.xml

```diff
@@ -2089,10 +2090,10 @@ secfrac = "." 1*6DIGIT</programlisting>
         </mediaobject>
       </figure>
     </section>
-    <section xml:id="_refCloudRecordingEncryption">
-      <title>Encryption</title>
+    <section xml:id="_refCloudRecordingFrameEncryption">
+      <title>Frame encryption</title>
       <para>
-        A device signaling support for recording to an external target with encryption shall support writing encrypted files according to ISO/IEC 23001-7 (common encryption in ISO base media file format files).
+        A device signaling support for recording to an external target with frame encryption shall support creating encrypted files according to ISO/IEC 23001-7 (common encryption in ISO base media file format files).
```

Comment:
ONVIF/ISO do not OR should not deter devices from using encryption either for local recordings or recordings pushed to external targets and hence I would suggest below 

```suggestion
        A device signaling support for recording with frame encryption via EncryptionEntryLimit shall support creating encrypted files according to ISO/IEC 23001-7 (common encryption in ISO base media file format files).
```